multinet/__init__.py

```diff
@@ -37,8 +72,9 @@ def create_app(config: Optional[MutableMapping] = None) -> Flask:
     if config is not None:
         app.config.update(config)
 
-    allowed_origins = get_allowed_origins()
-    CORS(app, origins=allowed_origins, supports_credentials=True)
+    app.config["ALLOWED_ORIGINS"] = get_allowed_origins()
+
+    register_cors_func(app, True)
```

Comment:
In keeping with retaining the old calling sequence:

```suggestion
    register_cors_func(app, with_credentials = True)
```